debian/additions/mariadb.conf.d/60-galera.cnf

```diff
@@ -1,21 +1,24 @@
 #
 # * Galera-related settings
 #
-# See the examples of server wsrep.cnf files in /usr/share/mariadb
-# and read more at https://mariadb.com/kb/en/galera-cluster/
+# See the examples of server wsrep.cnf files in /usr/share/mariadb and read
+# more at https://mariadb.com/kb/en/galera-cluster/ and
+# https://mariadb.com/kb/en/configuring-mariadb-galera-cluster/
 
 [galera]
-# Mandatory settings
-#wsrep_on                 = ON
 #wsrep_cluster_name       = "MariaDB Galera Cluster"
+
+# Mandatory settings
+#wsrep_provider           = /usr/lib/galera/libgalera_smm.so
 #wsrep_cluster_address    = gcomm://
-#binlog_format            = row
+#binlog_format            = ROW
+#wsrep_on                 = ON
 #default_storage_engine   = InnoDB
-#innodb_autoinc_lock_mode = 2
 
 # Allow server to accept connections on all interfaces.
 #bind-address = 0.0.0.0
 
-# Optional settings
-#wsrep_slave_threads = 1
+# Performance related settings
+#innodb_autoinc_lock_mode = 2
 #innodb_flush_log_at_trx_commit = 0
+#wsrep_slave_threads = 1
```

Comment:
so why is this only 1? To get any benefit it has to be more than one.

Reply:
The default is 1. Our docs also point out that "if your cluster nodes seem to have frequent consistency problems, then setting the value to 1 will probably fix the problem.", hopefully only referencing some old issue! Even the doc at https://galeracluster.com/library/documentation/mysql-wsrep-options.html#wsrep-applier-threads (which replaces wsrep-slave-threads in the latest version) defaults to 1, and only gives very loose suggestions for changing.

Reply:
So, can we decide what to do about it?

Reply:
@fauust set this to 8 in the comment please. Then ok to merge.